internal/statusupdater/statusupdater.go

```diff
@@ -64,6 +69,34 @@ func (s *statusUpdater) UpdateModuleStatus(ctx context.Context,
 	return s.client.Status().Update(ctx, mod)
 }
 
+func (s *statusUpdater) PresetPreflightVerificationStatus(ctx context.Context,
```

Comment:
I don't see any usage of that method so I am guessing one will call it with?
```
PresetPreflightVerificationStatus(ctx, &pv, &pv.status)
```

If this is the case, why do we need both `pv` and `status` as arguments?

Same applied for the other methods in this package.

Reply:
This function will be called from the preflight package ( PRs for that one will follow).
pv is the whole PreflightValidation, and CRStatus(ps) is just on entry in the array of CRStatuses that comprise the PreflightValidationStatus (see api/v1alpha1/preflightvalidation_types.go ) 
The calling method already obtains the needed CRStatus entry , since it needs it for its own logic, so i don't see the reason not to pass it as an paramter to the functions